test_data/yast/installation/network_config_s390x_svirt.yaml

```diff
@@ -0,0 +1,30 @@
+---
+network:
+  network_manager: wicked
+  device: eth0
+  config_files:
+    - path: /etc/sysconfig/network/ifcfg-eth0
+      entries:
+        - BOOTPROTO='dhcp'
```

Comment:
Hmm, I wonder if we need to keep the key-pair approach we use in kdump or just use the way to check complete lines? As here we could have used mapping. WDYT?